src/omg-cli/omg/tiles/gcp_director/configuration.go

```diff
@@ -106,13 +106,6 @@ func resources(envConfig *config.EnvConfig) commands.ResourceConfiguration {
 	var instanceCount *int
 	var instanceType commands.CompilationInstanceType
 
-	if envConfig.SmallFootprint {
```

Comment:
we need to restrict Small Footprint so it doesn't eat up quota beyond non-default. Is this necessary to install?

Reply:
I can restrict to 1 instance. But need a large instance instead of small. compilation of health watch used up 2 hours in small while large only takes around 10 minutes

Reply:
oh wow! yeah that's unacceptable. go with a single large for now and I'll tweak it if we need it. Cores are the big limitation so we can always toss memory with the same cores (eg large.mem, medium.mem) .